src/router/handlers/admin.js

```diff
@@ -0,0 +1,356 @@
+/**
+ * @swagger
+ * /org/:
+ *  post:
+ *    description: Use to create an organization
+ *    parameters:
+ *      - name: ha-api-token
+ *        in: header
+ *        required: true
+ *        schema:
+ *          type: string
+ *          format: uuid
+ *      - name: organization
+ *        in: body
+ *        required: true
+ *        schema:
+ *          type: object
+ *    responses:
+ *      '201':
+ *        description: 'Success'
+ *      '400':
+ *        description: 'Invalid syntax'
+ *      '401':
+ *        description: 'JWT not found in header'
+ *      '403':
+ *        description: 'JWT does not have admin privileges'
+ */
+let createOrganization = async (req, res) => {
+
+};
+
+/**
+ * @swagger
+ * /org/:
+ *  put:
+ *    description: Use to update an organization
+ *    parameters:
+ *      - name: ha-api-token
+ *        in: header
+ *        required: true
+ *        schema:
+ *          type: string
+ *          format: uuid
+ *      - name: organization
+ *        in: body
+ *        required: true
+ *        schema:
+ *          type: object
+ *    responses:
+ *      '200':
+ *        description: Success
+ *      '400':
+ *        description: 'Invalid syntax'
+ *      '401':
+ *        description: 'JWT not found in header'
+ *      '403':
+ *        description: 'JWT does not have admin privileges'
+ *      '404':
+ *        description: 'Not found'
+ */
+let updateOrganization = async (req, res) => {
+
+};
+
+/**
+ * @swagger
+ * /hacks/:
+ *  post:
+ *    description: Use to create a hackathon
+ *    parameters:
+ *      - name: ha-api-token
+ *        in: header
+ *        required: true
+ *        schema:
+ *          type: string
+ *          format: uuid
+ *      - name: hackathon
+ *        in: body
+ *        required: true
+ *        schema:
+ *          type: object
+ *    responses:
+ *      '201':
+ *        description: 'Success'
+ *      '400':
+ *        description: 'Invalid syntax'
+ *      '401':
+ *        description: 'JWT not found in header'
+ *      '403':
+ *        description: 'JWT does not have admin privileges'
+ */
+let createHackathon = async (req, res) => {
+
+};
+
+
+/**
+ * @swagger
+ * /hacks/:hid/:
+ *  put:
+ *    description: Use to update a hackathon
+ *    parameters:
+ *      - name: ha-api-token
+ *        in: header
+ *        required: true
+ *        schema:
+ *          type: string
+ *          format: uuid
+ *      - name: hackathon
+ *        in: body
+ *        required: true
+ *        schema:
+ *          type: object
```

Comment:
I was curious how this would be handled in the docs—just calling it an object is a safe move. Later on, is there a way to add an example to the documentation to show what such an object might look like (with all its properties)?

Reply:
Yeah, exactly! Once we narrow down the exact params I'll include them here.